opm/simulators/wells/WellAssemble.hpp

```diff
@@ -21,8 +21,8 @@
 */
 
 
-#ifndef OPM_WELLINTERFACE_EVAL_HEADER_INCLUDED
-#define OPM_WELLINTERFACE_EVAL_HEADER_INCLUDED
+#ifndef OPM_ASSEMBLE_HEADER_INCLUDED
+#define OPM_ASSEMBLE_HEADER_INCLUDED
```

Comment:
Tiny nit: Maybe include the word `WELL` here to reduce the risk of conflicting symbols in case we ever get a generalised assembly/assemble component in the future?

Reply:
sure.